src/vstart.sh

```diff
@@ -177,6 +177,8 @@ io_uring_enabled=0
 with_jaeger=0
 force_addr=0
 osds_per_host=0
+require_osd_and_client_version=""
```

Comment:
`usage` as well even without any elaborated descriptions, just to inform about theirs presence.